src/index.html

```diff
@@ -22,6 +22,425 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item first_item">
+            <a
+              class="is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav__item"
+            data-qa="hover"
+          >
+            <a
+              class="link"
+              href="#laptops-and-computers"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__content">
+          <img
+            class="card__photo"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          />
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            Reviews: 5
+          </div>
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+
+            <div class="card__price-tag">$2,199</div>
```

Comment:
You need to use semantic tags for text everywhere
```suggestion
            <p class="card__price-label">Price:</p>

            <p class="card__price-tag">$2,199</p>
```